src/Components/Auth/Login.tsx

```diff
@@ -146,26 +141,24 @@ export const Login = (props: { forgot?: boolean }) => {
     return form;
   };
 
-  const handleForgetSubmit = (e: any) => {
+  const handleForgetSubmit = async (e: any) => {
     e.preventDefault();
     const valid = validateForgetData();
     if (valid) {
       setLoading(true);
-      dispatch(postForgotPassword(valid)).then((resp: any) => {
-        setLoading(false);
-        const res = resp && resp.data;
-        if (res && res.status === "OK") {
-          Notification.Success({
-            msg: t("password_sent"),
-          });
-        } else if (res && res.data) {
-          setErrors(res.data);
-        } else {
-          Notification.Error({
-            msg: t("something_wrong"),
-          });
-        }
-      });
+      const { res, error } = await request(routes.forgotPassword);
+      setLoading(false);
+      if (res && res.statusText === "OK") {
+        Notification.Success({
+          msg: t("password_sent"),
+        });
+      } else if (res && error) {
+        setErrors(error);
+      } else {
+        Notification.Error({
+          msg: t("something_wrong"),
+        });
```

Comment:
5xx something went wrong error notifications are handled by the new `request` module. You may skip that.

Reply:
yaa okay